administration/src/Logout.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { ReactElement, useContext } from 'react'
+import { Navigate } from 'react-router'
+
+import { AuthContext } from './AuthProvider'
+
+export const Logout = (): ReactElement => {
+  const { signOut } = useContext(AuthContext)
+
+  signOut()
```

Comment:
🔧 i think you should use a `useEffect` here to avoid side effects die to rerendering

```
  useEffect(() => {
    signOut()
  }, [signOut])

```